packages/CustomRecordViewButton/readme.MD

```diff
@@ -0,0 +1,6 @@
+#Custom record view button
+
+This module creates a button that is displayed on an Accounts record.  When a user clicks the button, the system 
+displays a simple alert dialog.
+
+You MUST run a Quick Repair and Rebuild (can be found under Administration > Repair) in order for the button to work.
```

Comment:
We could add a post install/uninstall script that runs the equivalent repair so that Admins do not need to do this manually. See https://github.com/sugarcrm/BuildingBlocks/blob/master/packages/ScriptLoader/ for an example.